litt/disktable/disk_table.go

```diff
@@ -574,6 +574,71 @@ func (d *DiskTable) Get(key []byte) ([]byte, bool, error) {
 	return data, true, nil
 }
 
+func (d *DiskTable) CacheAwareGet(
+	key []byte,
+	onlyReadFromCache bool,
+) (value []byte, exists bool, hot bool, err error) {
+
+	if ok, err := d.fatalErrorHandler.IsOk(); !ok {
+		return nil, false, false, fmt.Errorf(
+			"Cannot process Get() request, DB is in panicked state due to error: %w", err)
```

Comment:
outdated error message, method is called `CacheAwareGet`

Reply:
fixed?